tests/grf/test_solver.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+import numpy.testing as npt
+import pytest
+
+import glass.grf
+
+
+@pytest.fixture
+def cl():
+    lmax = 100
+    ell = np.arange(lmax + 1)
+    return 1e-2 / (2 * ell + 1) ** 2
```

Comment:
Can we move this to `conftest.py`? We have all the fixtures in one place for the ease of access/maintainability.

Reply:
This isn't really applicable to anything outside this module. I'd rather move the three lines into every test than move the fixture to conftest.py.

Reply:
But this is now inconsistent with the rest of the code.

Reply:
Do you mean that other fixtures are in conftest? That's fine, this fixture is not applicable to any other code, it should not be available globally.

Reply:
Yeah I did mean that. Let's leave it